rollup.config.js

```diff
@@ -1,6 +1,5 @@
-const flow = require('rollup-plugin-flow');
 const babel = require('rollup-plugin-babel');
-const uglify = require('rollup-plugin-uglify');
+const { uglify } = require('rollup-plugin-uglify');
```

Comment:
This new in version > 6?

Reply:
Yop, they seem to have changed their API